recipe/meta.yaml

```diff
@@ -37,11 +37,12 @@ outputs:
         - prettytable
         - urdfdom-py
         - array-api-compat
+        - mujoco
```

Comment:
This dependency match the non-optional dependencies in https://github.com/ami-iit/adam/blob/219991c6c646ce0c6f1aa1251279e7906d7b4753/pyproject.toml#L28-L35 . As `mujoco` is not there, probably we can avoid to add it also here?